content/docs/1.1.0/high-availability/node-failure.md

```diff
@@ -55,6 +55,22 @@ Which then allows the pending replacement pods to start correctly with the reque
 Longhorn will recover the Volume Attachment from a failed node as soon as there are pending replacement pods available.
 Which then allows the pending replacement pods to start correctly with the requested volumes being available.
 
+### Pod Deletion Policy When Node Downs
+
+Kubernetes never force deletes pods of StatefulSet or Deployment on a downed node. Since the pod on the down node wasn't removed,
```

Comment:
Always use `downed node` or `down node`

Reply:
Thank you. Fixed

Reply:
I don't think it should be downed node. Failed node sounds better to me

Reply:
"when node downs" can't be used because downs isn't a verb